.gitignore

```diff
@@ -138,3 +138,9 @@ dmypy.json
 client_secrets.json
 credentials.json
 settings.yaml
+
+# Logs
+log/
```

Comment:
This directory is full of mess. Do we really need all the files?

Reply:
This may seem mess, but these files are tensorboard data. Don’t tinker with them directly, tensorboard creates nice visualizations from them.

Ofc you should push only models and logs that are worth saving.

Reply:
Since logs contain some very big data, I will leave it in gitignore for now